src/Core/Session/Factory/Session.php

```diff
@@ -82,10 +80,15 @@ public function createSession(App\Mode $mode, App\BaseURL $baseURL, IManageConfi
 							$handler = new Handler\Cache($cache, $logger);
 						}
 						break;
+					default:
+						$handler = null;
 				}
 
 				$session = new Type\Native($baseURL, $handler);
 			}
+		} catch (\Throwable $e) {
+			$logger->notice('Unable to create session', ['mode' => $mode, 'session_handler' => $session_handler, 'exception' => $e]);
+			$session = new Type\Memory();
```

Comment:
```suggestion
			$session = new self::HANDLER_DEFAULT();
```

Reply:
This will try to instantiate an object of the class `Friendica\Core\Session\Factory\database`, not sure it's going to work.